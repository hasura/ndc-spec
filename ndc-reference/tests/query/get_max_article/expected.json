[
  {
    "rows": [
      {
        "__value": {
          "id": 3,
          "title": "The Design And Implementation Of Programming Languages"
        }
      }
    ]
  }
]