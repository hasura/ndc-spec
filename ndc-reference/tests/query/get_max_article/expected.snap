---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/get_max_article/request.json
---
[
  {
    "rows": [
      {
        "__value": {
          "id": 3,
          "title": "The Design And Implementation Of Programming Languages"
        }
      }
    ]
  }
]
